static/app/views/performance/browser/webVitals/components/performanceScoreRing.tsx

```diff
@@ -87,7 +87,7 @@ function PerformanceScoreRing({
       currentRotate += (360 * maxValue) / sumMaxValues;
 
       const cx = radius + barWidth / 2;
-      const key = `${cx}-${backgroundColors[index]}`;
+      const key = `${index}-${cx}-${backgroundColors[index]}`;
```

Comment:
Fine since it's only 5 elements, but I think this would make more sense if the `values` for the ring were objects like `{label: 'FCP', value: 12, maxValue: 30}` and you could use `label` as a stable key instead of the loop index

Reply:
I went ahead and refactored `values` as suggested, with the webvital names as keys!

Reply:
Big play!